packages/manager/modules/server-sidebar/src/telecom.constants.js

```diff
@@ -0,0 +1,368 @@
+  title: 'number',
+  icon: 'ovh-font ovh-font-hashtag',
+  state: 'telecom.orders.alias',
+  app: [TELECOM],
+  feature: 'telephony',
+};
+
+export const DOMAIN_ORDER_CONFIG = {
+  id: 'domain',
+  title: 'domain',
+  icon: 'ovh-font ovh-font-domain',
+  linkId: 'domain',
+  target: '_blank',
+  external: true,
+  onClick: 'orders::domain-name::order',
+  app: [TELECOM],
+};
+
+export const PACK_ORDER_CONFIG = {
+  id: 'internet',
+  title: 'internet',
+  icon: 'ovh-font ovh-font-telecom-ethernet',
+  feature: 'pack',
+  children: [
+    {
+      title: 'internet_xdsl',
+      linkId: 'internet_xdsl',
+      target: '_blank',
+      external: true,
+      onClick: 'orders::xdsl::order',
+    },
+    {
+      title: 'internet_fiber',
+      linkId: 'internet_fiber',
+      target: '_blank',
+      external: true,
+      onClick: 'orders::fibre::order',
+    },
+    {
+      title: 'internet_sdsl',
+      linkId: 'internet_sdsl',
+      target: '_blank',
+      external: true,
+      onClick: 'orders::sdsl::order',
+    },
+    {
+      title: 'internet_adsl_creation',
+      linkId: 'internet_adsl_creation',
+      target: '_blank',
+      external: true,
+      onClick: 'orders::adsl-new::order',
+    },
+    {
+      title: 'internet_otb',
+      linkId: 'internet_otb',
+      target: '_blank',
+      external: true,
+    },
+  ],
+};
+
+export const TELEPHONY_ORDER_CONFIG = {
+  id: 'telephony',
+  title: 'telephony',
+  icon: 'ovh-font ovh-font-phone',
+  app: [TELECOM],
+  feature: 'telephony',
+  children: [
+    {
+      title: 'telephony_voip',
+      linkId: 'telephony_voip',
+      target: '_blank',
+      external: true,
+      onClick: 'orders::telephony::voip::order',
+      app: [TELECOM],
+    },
+    {
+      title: 'telephony_siptrunk',
+      linkId: 'telephony_siptrunk',
+      target: '_blank',
+      external: true,
+      onClick: 'orders::telephony::sip-trunk::order',
+      app: [TELECOM],
+    },
+    {
+      title: 'telephony_siptrunk_call',
+      linkId: 'telephony_siptrunkCall',
+      target: '_blank',
+      external: true,
+      onClick: 'orders::telephony::sip-trunk-included::order',
+      app: [TELECOM],
+    },
+    {
+      title: 'telephony_accessories',
+      state: 'telecom.orders.accessories',
+      app: [TELECOM],
+    },
+  ],
+};
+
+export const EMAIL_ORDER_CONFIG = {
+  id: 'email',
+  title: 'email',
+  icon: 'ovh-font ovh-font-mail',
+  app: [TELECOM],
+  children: [
+    {
+      title: 'email_exchange',
+      linkId: 'email_exchange',
+      target: '_blank',
+      external: true,
+      onClick: 'orders::email::email-microsoft-exchange::order',
+      app: [TELECOM],
+    },
+    {
+      title: 'email_sharepoint',
+      linkId: 'email_sharepoint',
+      target: '_blank',
+      external: true,
+      tracker: 'orders::email::microsoft-sharepoint::order',
+      app: [TELECOM],
+    },
+  ],
+};
+
+export const OFFICE_ORDER_CONFIG = {
+  id: 'office365',
+  title: 'office365',
+  svg:
```

Comment:
**nitpick(blocking):** We have to use following class `ms-Icon ms-Icon--OfficeLogo`.

Reply:
Ok, just reused the existing code but will update it.